edx_notifications/digest_timers.py

```diff
@@ -0,0 +1,35 @@
+from contracts import contract
+import datetime
+from django.dispatch import receiver
+from edx_notifications.data import NotificationMessage, NotificationCallbackTimer
+from edx_notifications.signals import perform_timer_registrations
+from edx_notifications.stores.store import notification_store
+
+DAILY_DIGEST_TIMER_NAME = 'daily-digest-timer'
+WEEKLY_DIGEST_TIMER_NAME = 'weekly-digest-timer'
+
+
+@receiver(perform_timer_registrations)
+def register_digest_timers():
+    store = notification_store()
+
+    weekly_digest_timer = NotificationCallbackTimer(
```

Comment:
Can you first look to see if this timer name already exists? I think if we don't do this, every time the server restarts, it's going to reset the timer. There's a method that I think is called store.get_notification_timer(timer_name).